lib/article_json/export/common/html/shared/module_utils.rb

```diff
@@ -0,0 +1,24 @@
+module ArticleJSON
+  module Export
+    module Common
+      module HTML
+        module Shared
+          module ModuleUtils
```

Comment:
I don't like having this in a separate module much. Took me quite a bit to understand what's going on and what the name `ModuleUtils` actually means :)

I would just leave it in the exporter module where it s used, much easier to grasp than a module loading another module that extends the base class :)

Reply:
The issue is that this is used in two modules: In the `Exporter` as well as in the `Elements::Base`. I first had the method duplicated in both modules, but that also didn't look right. Any better implementation ideas or naming suggestions?

Reply:
Like discussed in person, better to define explicit `namespace` methods where needed...